control-plane-operator/controllers/hostedcontrolplane/oauth/pdb.go

```diff
@@ -0,0 +1,29 @@
+package oauth
+
+import (
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	policyv1 "k8s.io/api/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func ReconcilePodDisruptionBudget(pdb *policyv1.PodDisruptionBudget, p *OAuthServerParams) error {
```

Comment:
Nit: Seems like we could DRY out all the PDB reconcile func into something common that takes the label selector and the availability policy as args

Reply:
Agree, figured a pattern would start to emerge eventually.

Reply:
I don't think I want to do it in this PR, though.